objects.js

```diff
@@ -8520,6 +8523,48 @@ WatcherMorph.prototype.userMenu = function () {
     return menu;
 };
 
+WatcherMorph.prototype.valueExporter = function (format) {
+    var myself = this,
+        value = this.currentValue,
+        contents,
+        format = format || 'plain',
+        ide = myself.parentThatIsA(IDE_Morph);
+
+    return function () {
+        switch (format) {
+            case 'plain':
+                contents = value instanceof List ? 
+                    value.asArray().join(', ') : 
```

Comment:
But that absolutely depends on your use case!

If your list is a list of temperatures `[23, 22.4, 25, 30]`, then you're right, it's a single column. But if your list represents a person `['Bernat', 32, 'Barcelona']`, then you're wrong, it's a single row.

Which one is easier to deal with depends again on your use case. Anyone can search and replace commas, but not everyone knows how to `%s/\n/;/g`.

Anyway, I see I'm the only comma advocate, so let me change it. :angry: 

That is, until a fourth person joins the conversation and asks for semicolons.